### DISCUSSION TITLE
Getting white page on expenses page

### DISCUSSION THREAD

**Discussion by bbdata** [firefly-iii/firefly-iii#8763] [Firefly III]
I setup an install via CPanel, everything was working okay up til after a data import.

Now when I visit the expenses page (/transactions/withdrawal) it leads to white page.

I turned on debugging and looked at the storage/logs ... no error is recorded.

Any pointers?

**Comment by JC5** [comment]
Is it only that specific page that gives the error?

**Comment by bbdata** [comment]
Yes. The expenses page is the only one that does this. Rest of the app is
working fine.

**Reply by JC5** [reply]
I suspect that one transaction is broken. You may have to try to find it by browsing to /transactions/show/xxxx until you find it, going backwards or forwards.

This is kind of annoying but may be the only way to find it.

Incidentally, are there any account pages (from /accounts/asset) that are also broken?

**Reply by bbdata** [reply]
Thanks for the recommendation. I'll see what I can uncover.

Oddly, none of the account page are broken. They load fine.
